<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScoresByShooter</class>
 <widget class="QMainWindow" name="ScoresByShooter">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>915</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>View Scores By Shooter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="comboSBSDate">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>35</y>
      <width>136</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtShowScores">
    <property name="geometry">
     <rect>
      <x>525</x>
      <y>35</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Show Scores</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>80</y>
      <width>876</width>
      <height>466</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTableWidget" name="tblScores">
       <column>
        <property name="text">
         <string>Date</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Sess</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Name</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>EVENT</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>DIVISION</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>CAL</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>SCORE</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="lblSession">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>5</y>
      <width>896</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lnShooter">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>34</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QComboBox" name="comboSession">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>35</y>
      <width>86</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>915</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>comboSBSDate</tabstop>
  <tabstop>comboSession</tabstop>
  <tabstop>lnShooter</tabstop>
  <tabstop>ButtShowScores</tabstop>
  <tabstop>tblScores</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
